src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_goes_through_all_retries.cs

```diff
@@ -0,0 +1,176 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Config;
+    using NServiceBus.Pipeline;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_goes_through_all_retries : NServiceBusAcceptanceTest
```

Comment:
The point is to test if moving the message to error queue or deferring it by SLR does not accidentally cause outgoing messages to be sent. So these tests verify if retries don't have unexpected effects on transactions handling.